### PR TITLE
wxGUI: Fixed E722: bare 'except' in iscatt/

### PR DESCRIPTION
**Pull request by arohanajit**
Specified `except` clause Exceptions to fix `E722` in `iscatt\`

### CODE REVIEW

File: gui/wxpython/iscatt/frame.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def CursorPlotMove(self, x, y, scatt_id):
             x = int(round(x))
             y = int(round(y))
             coords = True
-        except:
+        except (ValueError, TypeError):
```
Comment (petrasovaa) [inline gui/wxpython/iscatt/frame.py:252]: I don't see how valueError could be raised?

Comment (arohanajit) [reply gui/wxpython/iscatt/frame.py:252]: In NaN of 'inf' values which can be rounded but not converted to int

Comment (petrasovaa) [reply gui/wxpython/iscatt/frame.py:252]: Good point, but I would keep TypeError only, Nan and inf are very unlikely here. Looking at the code, the coordinates can be None, that still results in TypeError.

**Review by petrasovaa — APPROVED** [review]

**Review by petrasovaa — CHANGES_REQUESTED** [review]
There are conflicts

**Review by petrasovaa — APPROVED** [review]

**Comment by petrasovaa** [comment]
@arohanajit this needs a fix, `struct_Range` is in grass.lib.raster and `struct_Cell_head` in grass.lib.gis

**Pull request merged**